app/views/wise_transfers/new.html.erb

```diff
@@ -0,0 +1,110 @@
+<% disabled = !policy(@event).create_transfer? %>
+
+<% title "Send a Wise transfer" %>
+<%= render "events/nav", selected: :transfers %>
+<% page_sm %>
+
+<% if @event.present? && !policy(@event).create_transfer? %>
+  <%= render partial: "events/unauthorized_callout", locals: { action: "send a Wise transfer" } %>
+<% end %>
+
+<h1>Send a Wise transfer</h1>
+
+<%= render "callout",
+    type: "info",
+    icon: "wise",
+    title: "Important information about Wise transfers",
+    class: "!border-[#9FE870] [&>div.flex]:text-[#163300] dark:[&>div.flex]:text-[#9FE870]",
+    footer: link_to("Learn more about Wise on HCB →", "https://help.hcb.hackclub.com/article/65-wise") do %>
+  HCB is partnering with Wise to offer international transfers to over 100 countries. HCB imposes no fees or minimums, but any fees charged by Wise will be passed on to your organization.
+<% end %>
```

Comment:
I think we should mention here that we support fee-free international transfers over $500

Reply:
Sorry - to clarify, I think we should always charge on the fee for Wise transfers. My original comment would have been better phrased as: 

> I think we should mention here that we support fee-free wire transfers over $500

Reply:
Ah, I see. Yeah, that's fair. I'm not sure what the best way of doing this is, so I'm open to ideas. Our transfer options are a lot more complicated now than they used to be...